docs/whatisk8s.md

```diff
@@ -0,0 +1,40 @@
+
+# What is Kubernetes?
+
+Kubernetes is an open source container cluster orchestration platform that was started by Google in 2014.
+
+Use Kubernetes to manage your containerized applications across multiple hosts through automated deployment, container life-cycle maintenance, and application scaling.
+
+With Kuberenets, you are able to quickly and efficiently respond to customer demand: 
+
+ - Scale your applications on the fly
+ - Seamlessly roll out new features
+ - Maintain optimal hardware use (only use the resouces you need)
```

Comment:
Optimize use of your hardware by using only the resources you need

(grammar + typo)